.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,52 @@
+{
+	"name": "Eraser",
+	"image": "ghcr.io/cse-labs/k3d:latest",
+
+	// do not change these if you want Docker in Docker support
+	"runArgs": ["--init", "--privileged"],
+	"mounts": [ "source=/var/run/docker.sock,target=/var/run/docker-host.sock,type=bind" ],
+	"overrideCommand": false,
+
+	// some base images require a specific user name
+	"remoteUser": "vscode",
+
+	// Set container specific settings
+	"settings": {
+		"files.trimTrailingWhitespace": true,
+		"files.trimFinalNewlines": true,
+		"files.insertFinalNewline": true
+	},
+
+	"hostRequirements": {
+		"cpus": 4
+	},
+
+	// forward ports for the app
+	"forwardPorts": [
+	],
+
+	// add labels
+	"portsAttributes": {
+	},
+
+	// Install extensions
+	"extensions": [
+	 	"ms-azuretools.vscode-docker",
+	 	"ms-kubernetes-tools.vscode-kubernetes-tools",
+	 	"davidanson.vscode-markdownlint",
+	 	"gruntfuggly.todo-tree",
+	 	"humao.rest-client",
+		"ms-vscode.makefile-tools"
```

Comment:
Addition of Golang may be a good add to this extension list Extension Identifier: `
golang.go`

Reply:
Added requested extension and added gopls installation command to post-create.sh